roles/apt-repos/defaults/main.yml

```diff
@@ -12,6 +12,9 @@ apt_repos:
   hwraid:
     repo: 'http://hwraid.le-vert.net/ubuntu'
     key_url: 'http://hwraid.le-vert.net/debian/hwraid.le-vert.net.gpg.key'
+  tkraid:
+    repo: 'http://archive.thomas-krenn.com/packages'
+    key_url: 'http://archive.thomas-krenn.com/tk-archive.gpg.pub'
```

Comment:
Any chance we could mirror it instead of adding yet more external resources that we have no control over?